<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TournamentPage</class>
 <widget class="QWidget" name="TournamentPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tournament Page</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="headerLayout">
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Турнир</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="yearLabel">
       <property name="text">
        <string>Год проведения</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="locationLabel">
       <property name="text">
        <string>Страна</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="searchLayout">
     <item>
      <widget class="QLineEdit" name="searchEdit">
       <property name="placeholderText">
        <string>Введите название турнира</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string>Найти</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="navigationLayout">
     <item>
      <widget class="QPushButton" name="prevButton">
       <property name="text">
        <string>&lt;-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="text">
        <string>-&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="actionsLayout">
     <item>
      <widget class="QPushButton" name="clubsButton">
       <property name="text">
        <string>Турнирная таблица</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="matchesButton">
       <property name="text">
        <string>Матчи</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="topScorersButton">
       <property name="text">
        <string>Таблица бомбардиров</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
